docs/UserGuide.md

```diff
@@ -93,69 +97,126 @@ Example:
 * The search will match substrings e.g `S912345` will match `S9123456A`
 * Search will return all records that matches any of the keywords for each field e.g `n/Alice Bob` will return the records of all patients with name containing `Alice` or `Bob`
 
-Examples: 
+####Examples: 
 * `find i/S9123456A` returns the patient record with IC `S9123456A`
 * `find n/Billy Alice a/Clementi i/S99` returns the patient records with Name containing `Billy` or `Alice`, Address containing `Clementi` and IC containing `S99`
 
 ### 3.7 Accessing a patient's profile : `access`
 
 Accesses a specified patient's profile in MediBook.
 
-Format: `access <index>`
+####Format: 
+
+`access INDEX`
 
 * Use the `list` command to return to the main list page
 * Each patient's profile consists of 3 sections: Personal Details, Medical Details and Medical Notes.
     * Personal Details section (top left): Shows the personal details of the patient, including their Name, IC, Date of Birth, and Phone Number, as well as their Email, Address, Height, Weight, BMI and Blood Type if available.
     * Medical Details section (bottom left): Shows medical details as tags separated into 3 categories (to be implemented), namely: Allergies, (ongoing) Treatments and (preexisting) Conditions.
     * Medical Notes section (right): A scrollable panel showing the list of medical notes recorded for the specified patient.
 
-### 3.8 Adding a medical note: `note`
+### 3.8 Adding a medical note: `addnote`
 
 Adds a medical note to a patient.
 
-As a doctor, you can add a medical note to a patient when viewing his/her profile. MediBook will automatically add the 
-to the patient displayed on the screen.
+As a doctor, you can add a medical note to your patient when viewing his/her profile to keep track of consultations
+with your patients. MediBook will automatically add the medical note to the patient displayed on your screen.
 
-Format: `note [d/DATE] n/NAME_OF_DOCTOR c/CONTENT_OF_MEDICAL_NOTE`
+####Format: 
 
-* You have to be on a patient's profile page to add a medical note. This is done by first `find`-ing the patient by IC
+`addnote [d/DATE] n/YOUR_NAME m/YOUR_MCR c/CONTENT_OF_MEDICAL_NOTE`
+
+* You have to be on a patient's profile page to add a medical note. You can do this by first `find`-ing the patient by IC
 when on the main list and then `access`-ing the patient's index on the filtered list.
+* `MCR` refers to your medical registration number. It is in the format 'M@####$', where @ is a letter/digit, # is a digit and $ is a letter.
+* You do not need to include your title (Dr) in your name.
 * For your convenience, the date field can be omitted. MediBook will automatically select today's date.
-* If you specify the date of the medical note, it has to be in the past or today.
+* If you specify the date of the medical note, it cannot be in the future.
+
+####Example:
+
+Context: while viewing the profile page of patient with IC 'S9123456A'
```

Comment:
Can this sound more user centric? i.e. If you are viewing the profile of a patient with the IC...

Reply:
Ok updated